lib/json_web_token.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class JsonWebToken
+  class << self
+    def encode(payload, exp = 5.minutes.from_now)
+      payload[:exp] = exp.to_i
+      JWT.encode(payload, jwt_secret)
+    end
+
+    def decode(token)
+      body = JWT.decode(token, jwt_secret)[0]
+      HashWithIndifferentAccess.new body
+    rescue StandardError
```

Comment:
Should we be rescuing a specific kind of error here? This error handling would not give us any information about the specific JWT error.

Reply:
Didn't check the error type here because I don't return any special response if the token is expired vs. it just not being valid, both return a 403 response code. Is that fine?

Reply:
It may be worth distinguishing between the two for logging purposes. We could keep metrics on the different types of decode failure types.